lib/mumukit/platform/organization_mapping.rb

```diff
@@ -49,8 +65,17 @@ def self.map_organization_routes!(native, framework, &block)
       framework.configure_tenant_path_routes! native, &block
     end
 
+    def self.path_composition_for(request)
+      path_parts = Pathname(path_for(request)).each_filename.to_a
```

Comment:
just nitpicking, but you could avoid the `delete_at(0)` later by splitting here
`orga, *path_parts = Pathname(path_for(request)).each_filename.to_a`

Reply:
I'll change it as it's much cleaner that way